document/chapters/02-background.tex

```diff
@@ -929,14 +929,109 @@ \subsection{Signed Distance Fields}
+Passing Interface (MPI) \cite{Forum_1994}. The master process is responsible for
+initializing the problem, determining the load balance for the number of
+parallel processes requested by the user, and collecting/accumulating
+information at the end of the simulation. In MCNP, particle histories are
+divided into equal segments among the slave processes for evaluation. More
+information on the method for particle history division and random number
+sequence assignment was detailed by Deng and Xie \cite{Deng_1999}. For each
+slave process, a corresponding set of the nuclear data, tally data structures,
+and geometry representation are created. Duplication of the geometry
+representation has a significant implications for the use of CAD-based
+tessellations.
+
+\subsection{Impact on CAD-Base Geometry}
+
+The tessellations resulting from CAD geometries consume much more memory than
+corresponding CSG representations. This must be taken into consideration when
+planning parallel simulations clusters where the limitation for the number of
+processes per CPU can be limited by the memory used in a single process. Because
+DAGMC's CAD-based particle tracking is an addition to MCNP, it has no influence
+on the parallelism data model. Given that geometry representations are required
+to exist in each process of the parallel simulation, memory usage in DAGMC is
+closely monitored. Some of this memory usage is largely uncontrollable due to
+the reliance on CUBIT or Trelis' underlying tessellation algorithms,
+but additional data used to maintain geometric relationships and build acceleration data
+structures are kept to a minimum if possible.
+
+\begin{table}[H]
+    \centering
+  \begin{tabular}{l c c c}
+    \toprule
+    Model & Tessellation (MB) & Tessellation and MOAB BVH (MB) \\
+    \hline
+    FNG   & 92           & 168 \\
+    ATR   & 294          & 675 \\
+    UWNR  & 213          & 435 \\
+    nTOF  & 56           & 88  \\
+    \hline
+  \end{tabular}
+  \caption[Memory summary of performance benchmark models.]{A summary of the
+    memory usage for the performance benchmark models shown in Table
+    \ref{dag-mcnp-benchmarks} both with and without acceleration data structures.}
+  \label{tab:dag-mcnp-benchmarks-mem}
+\end{table}
+
+Table \ref{tab:dag-mcnp-benchmarks-mem} shows the memory usage for all of the
+benchmark models used to assess DAG-MCNP's performance relative to native MCNP
+geometry representations in Section \ref{sec:problem-statement}. All of the
+models consume relatively low amounts of memory compared to the storage found on
+many CPUs in High Performance Computing (HPC) environments, but the addition of
+acceleration data structures, in this instance MOAB's BVH of OBBs, can more
+than double the memory occupied for the geometry representation. This effect
+becomes even more pronounced in larger production models seen in Chapters
+\ref{ch:simd_bvh} and \ref{ch:high_valence}.
```

Comment:
Perhaps a sentence or two about the possibility of future shared memory implementations for on-node memory efficiency???